app/Http/Controllers/InteractiveFeatureExperiencesController.php

```diff
@@ -85,7 +85,7 @@ protected function show($slug)
     {
         if (in_array('kiosk', request()->segments())) {
             return redirect()->action(
-                'InteractiveFeatureExperiencesController@showKiosk',
+                [self::class, 'showKiosk'],
```

Comment:
Using the string version of the action was causing an error, but the array version doesn't 🤷 .